storage/mounts_test.go

```diff
@@ -180,7 +184,8 @@ func TestMountPoint_Mount(t *testing.T) {
 		dbutils.notebook.exit(mount_source)
 	`, mountName, expectedMountSource, expectedMountConfig)
 	testMountFuncHelper(t, func(mp MountPoint, mount Mount) (s string, e error) {
-		return mp.Mount(mount)
+		client := common.CommonEnvironmentClient()
```

Comment:
this test is a unit test - e.g. not talking to the real APIs. `common.CommonEnvironmentClient()` is making a real HTTP client, which may be calling azure cli / ~/.databrickscfg configured workspace. not desired, please change.

Reply:
will change this to use a mock client